mlflow/tracking/fluent.py

```diff
@@ -570,7 +571,7 @@ def get_parent_run(run_id: str) -> Optional[Run]:
     return MlflowClient().get_parent_run(run_id)
 
 
-def log_param(key: str, value: Any) -> Any:
+def log_param(key: str, value: Any, synchronous: Optional[bool] = True) -> Any:
```

Comment:
@chenmoneygithub @sagarsumant Nit, but I think the type hint for `synchronous` is incorrect. It should just be `bool`.

```suggestion
def log_param(key: str, value: Any, synchronous: bool = True) -> Any:
```